index.bs

```diff
@@ -29,9 +30,17 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
       text: abort; for: Document; url: abort-a-document
       text: create and initialize a Document object; url: initialise-the-document-object
       text: history handling behavior; url: history-handling-behavior
+      text: link; url: the-link-element
       text: prompt to unload; url: prompt-to-unload-a-document
       text: refused to allow the document to be unloaded; url: refused-to-allow-the-document-to-be-unloaded
       text: traverse the history; url: traverse-the-history
+    urlPrefix: semantics.html
+      for: link
+        text: href; url: attr-link-href
+        text: referrerpolicy; url: attr-link-referrerpolicy
```

Comment:
I _think_ this is the best way to achieve what I want, but let me know if you think there's a better way

Reply:
You should just be able to do `<{link/href}>` and `<{link/referrerpolicy}>`, without these. Same for `<{link}>`, without the above.